### PR TITLE
Blog Stats Widget: display message when no stats are displayed

### PR DESCRIPTION
**Pull request by leogermani**
For newly created sites, there's no Stats to be displayed in the Stats widgets, however the current message is misleading. The message suggests that there's an error with the widget.

This PR adds a new state to the widget, clarifying that there's nothing wrong, it's just that there is no stats to be displayed yet.

Before:
screen-shot-on-2021-04-09-at-12-28-21

After:
Captura de Tela 2021-04-09 às 16 51 38

#### Changes proposed in this Pull Request:

* Display a proper message in the Stats widget for new sites without stats

#### Jetpack product discussion

p9dueE-2EQ-p2#comment-4807

#### Does this pull request change what data or activity we track or use?

No

#### Testing instructions:

* Launch a new JN site
* Connect it
* Jetpack > Settings > Writing
* Enable widgets
* Go to Appearence > Widgets
* Add a Blog Stats widget
* Check the front end of the site and see the new message

If possible, also test this branch with an existing site with some stats to make sure the widget is still working as expected.

### CODE REVIEW

File: projects/plugins/jetpack/modules/widgets/blog-stats.php
Comment (jeherve) [inline projects/plugins/jetpack/modules/widgets/blog-stats.php:147]: I like the idea, that's a nice addition!

As for the wording, I'll let the experts weigh in, but if we need to improve it, maybe we should use something similar to the Top Posts widget?
https://github.com/Automattic/jetpack/blob/e13487d0dbbe4c743739b6f1a3e37c143a8c2614/projects/plugins/jetpack/modules/widgets/top-posts.php#L311

**Review by jeherve — APPROVED** [review]

File: projects/plugins/jetpack/modules/widgets/blog-stats.php
Comment (leogermani) [reply projects/plugins/jetpack/modules/widgets/blog-stats.php:147]: That's a good idea for when the current user is an admin! Nice

**Comment by carolynannewells** [comment]
Morning! Popping in from editorial. I would clarify this to say, "There are no stats to display yet. Please try again later."

**Comment by leogermani** [comment]
> Morning! Popping in from editorial. I would clarify this to say, "There are no stats to display yet. Please try again later."

Thanks for the input @carolynannewells. 

Just one comment to make sure you got the proper context. This is displayed as the widget content, in the public-facing site of the site, to all site visitors. It doesn't sound totally right to me to ask a visitor to "try again"... what do you think?

**Comment by jeherve** [comment]
I'll merge this for now, even if the wording may still be improved, this is already an improvement over the current behaviour.

**Pull request merged**
